### PR TITLE
Update column widths

### PR DESCRIPTION
**Pull request by RMacfarlane**

### CODE REVIEW

**Comment by queerviolet** [comment]
t1

**Comment by queerviolet** [comment]
t1

**Comment by queerviolet** [comment]
t2

**Comment by queerviolet** [comment]
t3

**Comment by queerviolet** [comment]
t4

**Comment by queerviolet** [comment]
t5

**Comment by queerviolet** [comment]
t6

**Comment by queerviolet** [comment]
t7

**Comment by queerviolet** [comment]
t8

File: views/cart.hbs [hunk 1]
```diff
@@ -4,14 +4,14 @@
         <ul class="list-group">
             {{# each products }}
             <li class="list-group-item clearfix">
-                <div class="col-xs-5">
+                <div class="col-xs-4">
                     <strong>{{ this.item.title }}</strong><br/>
                     <em>{{{this.item.description}}}</em>
                 </div>
-                <div class="col-xs-2 text-right">
```
Comment (queerviolet) [inline views/cart.hbs]: c1

Comment (queerviolet) [reply views/cart.hbs]: c2

File: views/cart.hbs [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
+                <div class="col-xs-4 text-right">
                     <span class="badge">{{ this.quantity }}</span>
                 </div>
-                <div class="col-xs-3 text-right">
```
Comment (queerviolet) [inline views/cart.hbs]: q2

File: views/cart.hbs [hunk 3]
```diff
@@ -4,14 +4,14 @@
         <ul class="list-group">
             {{# each products }}
             <li class="list-group-item clearfix">
-                <div class="col-xs-5">
```
Comment (queerviolet) [inline views/cart.hbs]: c1

Comment (queerviolet) [inline views/cart.hbs]: x1

Comment (RMacfarlane) [reply views/cart.hbs]: hello

**Pull request merged**